apps/bitwarden_mailconfig.sh

```diff
@@ -4,6 +4,7 @@
 
 # shellcheck disable=2034,2059
 true
+SCRIPT_NAME="Bitwarden mail-configuration"
```

Comment:
Bitwarden Mail Configuration

Please follow the same wording everywhere. 

Capital Capital Capital Letter

Thanks!